internal/pkg/cli/pipeline_show.go

```diff
@@ -0,0 +1,88 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package cli
+
+import (
+	"fmt"
+
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/workspace"
+	"github.com/spf13/cobra"
+)
+
+type showPipelineVars struct {
+	*GlobalOpts
```

Comment:
I believe we can have more than one pipeline per project, should this take an optional name flag?